numexpr/tests/test_numexpr.py

```diff
@@ -44,6 +51,15 @@
 MAX_THREADS = 16
 
 
+if not pytest_available:
+    def identity(f):
+        return f
+
+    pytest = MagicMock()
+    pytest.mark = MagicMock()
+    pytest.mark.thread_unsafe = identity
```

Comment:
It'd be better to register `thread_unsafe` also if `pytest` is installed but `pytest-run-parallel` isn't. That way the test suite can be run with `pytest` also without `pytest-run-parallel`